CONTRIBUTING.md

```diff
@@ -3,6 +3,62 @@
 We'd love to accept your patches and contributions to this project. There are
 just a few small guidelines you need to follow.
 
```

Comment:
[CLA](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#contributor-license-agreement) may need to be the first step here. What do you think?